pkg/storage/cacher_test.go

```diff
@@ -202,7 +202,6 @@ func TestWatch(t *testing.T) {
 	if err != nil {
 		t.Fatalf("Incorrect resourceVersion: %s", fooCreated.ResourceVersion)
 	}
```

Comment:
Since you are no longer changing initialVersion, please remove parsing it and use fooCreated.ResourceVersion instead